lib/composer.js

```diff
@@ -17,7 +17,7 @@ export default class Composer {
     this.configChange = atom.config.onDidChange('latex', () => {
       this.rebuildCompleted = new Set()
     })
-    this.checkEnvironment()
+    if (!atom.inSpecMode()) this.checkEnvironment()
```

Comment:
Since this is not an "exit junction" (a `return` statement or similar), let's wrap this one in braces and split it across multiple lines.